aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -74,6 +76,15 @@ module aptos_framework::account {
         recipient_address: address,
     }
 
+    /// This struct stores the challenge message that should be signed by the source account, when the source account
+    /// is delegating its rotation capability to the `recipient_address`.
```

Comment:
Would be good to quickly state in the comments how the V2 struct differs from V1.

Reply:
+1